parts/linux/cloud-init/artifacts/components.json

```diff
@@ -45,6 +55,20 @@
         }
       ]
     },
+    {
+      "downloadURL": "mcr.microsoft.com/oss/v2/kubernetes/metrics-server:*",
+      "amd64OnlyVersions": [],
+      "multiArchVersionsV2": [
+        {
+          "renovateTag": "registry=https://mcr.microsoft.com, name=oss/v2/kubernetes/metrics-server",
+          "latestVersion": "v0.7.2-3"
+        },
+        {
+          "renovateTag": "registry=https://mcr.microsoft.com, name=oss/v2/kubernetes/metrics-server",
+          "latestVersion": "v0.6.3-2"
```

Comment:
I think we had previously been caching an unused version of metrics-server until recently - are both of these versions absolutely necessary?

Reply:
I have tested these images on all AKS versions (1.27, 1.28, 1.29. 1.30 and 1.31) and will be rolling out these images to metrics-servers pods in AKS clusters.